node/src/reactor.rs

```diff
@@ -44,6 +44,7 @@ use std::{
     sync::atomic::Ordering,
 };
 
+use casper_types::EraId;
```

Comment:
Import in the wrong place

Reply:
Fixed in https://github.com/casper-network/casper-node/pull/2531/commits/9a4261fae4aa6c6834052ff12514d12d47c9e858